sktime/classification/shapelet_based/_rocket_classifier.py

```diff
@@ -28,14 +34,15 @@ class ROCKETClassifier(BaseClassifier):
     ----------
     num_kernels             : int, number of kernels for ROCKET transform
     (default=10,000)
-    ensemble                : boolean, create ensemble of ROCKET's (default=False)
-    ensemble_size           : int, size of the ensemble (default=25)
+    n_estimators            : int, ensemble size, optional (default=None)
```

Comment:
we should say that None means 1

Reply:
Updated documentation. Plus, I changed the execution to treat `n_estimators=1` exactly as `n_estimators=None`